storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { TextArea, TextAreaProps } from '@amsterdam/design-system-react'
+import { useArgs } from '@storybook/preview-api'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+type StoryProps = TextAreaProps & { invalid?: boolean }
+
+const meta = {
+  title: 'Components/Forms/Text Area',
+  component: TextArea,
+  args: {
+    defaultValue: paragraph,
```

Comment:
Using `defaultValue` makes the interaction a bit weird, using `value` is probably a better idea